v8/client/client.go

```diff
@@ -37,7 +37,7 @@ type Client struct {
 // Set the realm to empty string to use the default realm from config.
 func NewWithPassword(username, realm, password string, krb5conf *config.Config, settings ...func(*Settings)) *Client {
 	creds := credentials.New(username, realm)
-	return &Client{
+	cl := &Client{
```

Comment:
Is this change necessary?

Reply:
That was a left over from when I was logging stuff out to try figure out the issue. Good spot, it's been removed